r/rsparkling/R/h2o_context.R

```diff
@@ -37,20 +36,22 @@ h2o_flow <- function(sc) {
 #'
 #' @param sc Object of type \code{spark_connection}.
 #' @param x A \code{spark_dataframe}.
-#' @param name The name of the H2OFrame
+#' @param name The name of the H2OFrame.
+#' @param strict_version_check (Optional) Setting this to FALSE does not cross check version of H2O and attempts to connect.
 #' @export
-as_h2o_frame <- function(sc, x, name=NULL) {
+as_h2o_frame <- function(sc, x, name=NULL, strict_version_check=TRUE) {
```

Comment:
New argument is required for now because `as_h2o_frame` call `h2o.init()` in fixed version available in maven repo(?). At the moment it is sparkling water limitation AFAIK. Argument could be removed when we handle this better.